gitlab/fetcher.go

```diff
@@ -2,6 +2,9 @@ package gitlab
 
 import "github.com/xanzy/go-gitlab"
 
+// ReProjectName represents regular expression matching to a project name
+const ReProjectName = "^([^/]+(?:/[^/]+)+)"
```

Comment:
I prefer to `reProjectName` (private) than `ReProjectName` (public).

Because this is used in only `gitlab` package.

Reply:
It makes sense.

Fixed at 33b504a93654c0756c9585efa260da90205d8d17.